lisa/sut_orchestrator/libvirt/platform.py

```diff
@@ -89,21 +89,23 @@ class BaseLibvirtPlatform(Platform, IBaseLibvirtPlatform):
         StartStop,
     ]
 
+    # Static variables to handle multiple task_runners
+    # Port var are used  for port forwarding in case of Remote Host
+    # 49512 is the first available private port
+    _next_available_port: int = 49152
```

Comment:
I believe only single instance of each platform class is created for the entire LISA process lifetime. So, do you need to make these variables static?

Reply:
No, when combinator/concurrency is used, multiple task_runners are created. And each task_runner will create a new platform instance. Hence, it needs to be static.